data/ga/legislature/Matt-Hatchett-87ff7d22-1683-4c7f-a2c1-4233b80c2039.yml

```diff
@@ -0,0 +1,39 @@
+id: ocd-person/87ff7d22-1683-4c7f-a2c1-4233b80c2039
+name: Matt Hatchett
+given_name: Matt
+family_name: Hatchett
+email: matt.hatchett@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/hatchett-matt-781.jpg
+party:
+- name: Republican
```

Comment:
@sroomf this person was been in the house district 150 from January 2011 until 01/09/2023. So they may already have a file elsewhere.